cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/Language.kt

```diff
@@ -67,7 +67,7 @@ abstract class Language<T : LanguageFrontend> : Node() {
     @Transient
     open val simpleTypes: Map<String, Type> =
         mapOf(
```

Comment:
Do these default values really make any sense? Because any language is / needs to overwrite it anyway?

Reply:
Probably not. We can delete them

Reply:
I am fine with default values, do you think they are to distinct for most languages? For char byte and short I would expect these sizes to be reasonable defaults

Reply:
> I am fine with default values, do you think they are to distinct for most languages? For char byte and short I would expect these sizes to be reasonable defaults

They are overwritten anyway by the specific type in each supported language that we currently have.